caddyhttp/fastcgi/dialer.go

```diff
@@ -25,9 +25,31 @@ type persistentDialer struct {
 	sync.Mutex
 }
 
+func (p *persistentDialer) Equals(q *persistentDialer) bool {
```

Comment:
I thought it would be nice to have the comparison member function next to the struct. If you like it better in the test file I will move it there.